server/routes/classes.ts

```diff
@@ -1,9 +1,32 @@
 import express, { Request, Response } from 'express'
 import { Class } from '../db/models/class'
+import multer from 'multer';
+import { v4 as uuidv4 } from 'uuid';
+import { Storage } from '@google-cloud/storage';
 import mongoose from 'mongoose'
 
 const router = express.Router()
+const storage = new Storage({
+    keyFilename: './GoogleCloudKey.json',
+  });
 
+const bucketName = 'classes-bucket';
+const bucket = storage.bucket(bucketName);
+
+const upload = multer({
+    storage: multer.memoryStorage(),
+    limits: {
+        fileSize: 5 * 1024 * 1024, // 5 MB
```

Comment:
The videos are large files, more than 5 MB. To test the functionality several times, you can try it with a short video, but the application should be able to upload larger videos, at least for implementation, although later we only upload videos of a few MB.